### DISCUSSION TITLE
Ability to convert coordinates in Batch Editor

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5588] [Feature Requests]
The WorkBench has the ability to convert coordinates before import, but the forms should have that option built in as well.

**Comment by maxpatiiuk** [comment]
This might make more sense as part of batch editor rather than forms

**Comment by grantfitzsimmons** [comment]
If it can be done on both that would be the best solution

**Comment by maxpatiiuk** [comment]
For more context:
This speaks about changing the values in the lat1str, long1str, lat2str, and long2str fields only
It won't be changing the values in the latitude1, longitude1, latitude2 or longitude2 fields.
First set of fields are strings. Second are decimals - both are used interchangeably depending on the need.
Strings usually contain the value in the format it was originally entered in.

A likely use case for this feature in the context of a form/batch edit is to convert legacy/external data to consistent format
